lib/button.js

```diff
@@ -126,7 +126,7 @@ function Button(opts) {
 
       timeout = setTimeout(function() {
         if (this.isDown) {
-          this.emit("hold", err);
+          this.emit("hold");
```

Comment:
Unfortunately, it's a mixed collection of `new Date()` and `Date.now()` which will also have to be fixed, but in another patch